site/cypress/e2e/Cypress-UI/navigation_page.spec.js

```diff
@@ -170,6 +170,18 @@ describe('tests navigation buttons for each level of access', () => {
         );
         checkButtons('grades_released_lab', null, null);
     });
+
+    ['instructor', 'ta', 'student'].forEach((user) => {
+        it(`should show the correct alert message for team homework in case of ${user}`, () => {
```

Comment:
I think something like this would sound better
```suggestion
        it(`should show the correct alert message for team homework for ${user}`, () => {
```